audio/generaluser-soundfont/Portfile

```diff
@@ -18,14 +19,30 @@ long_description    ${description} Provides the GeneralUser SoundFont by S. Chri
                     footprint (less than 30 MB of RAM). Due to its clever, detailed sound programming, \
                     GeneralUser GS can sound as good or better than SoundFonts that are 2-3 times its size.
 homepage            http://www.schristiancollins.com/generaluser.php
-master_sites        https://dl.dropboxusercontent.com/u/8126161/
+master_sites        https://www.dropbox.com/s/4x27l49kxcwamp5/
 use_zip             yes
 
 distname            GeneralUser_GS_${version}
+
+# This line can be removed once backward compatibility with MacPorts
+# base versions prior to PR macports/macports-base#55 is not needed.
 worksrcdir          "GeneralUser GS ${version}"
```

Comment:
Why would this line no longer be necessary? We'd still be fetching the same thing, no?

Reply:
PR macports/macports-base#55 makes specifying custom `worksrcdir`s unnecessary.